lib/cdo/github.rb

```diff
@@ -92,6 +92,34 @@ def self.compare(base:, head:)
     response.commits.map(&:commit).map(&:message)
   end
 
+  # Octokit Documentation: http://octokit.github.io/octokit.rb/Octokit/Client/Commits.html#compare-instance_method
+  # @param base [String] The base branch to compare against.
+  # @param compare [String] The comparison branch to compare.
+  # @raise [Exception] From calling Octokit.compare.
+  # @return [Boolean] Whether compare is ahead of base, i.e., whether compare
+  #   has commits not in base.
+  def self.ahead?(base:, compare:)
+    base_sha = sha(base)
+    compare_sha = sha(compare)
+
+    response = Octokit.compare(REPO, base_sha, compare_sha)
```

Comment:
Why are we getting the hashes at all here?  Compare should accept branch names.  [From the docs:](https://developer.github.com/v3/repos/commits/#compare-two-commits)

> ```
> GET /repos/:owner/:repo/compare/:base...:head
> ```
> Both `:base` and `:head` must be branch names in `:repo`.